backend/tests/api-e2e/test_api_snapshot.py

```diff
@@ -13,6 +13,36 @@
 # whose state is not reset between tests.
 
 
+@pytest.mark.api
```

Comment:
please add also tests for the service and mock results from the API (you can take a look how it's done e.g in ```patch_etherscan_transactions_api```)

Reply:
:+1: